var/spack/repos/builtin/packages/rhash/package.py

```diff
@@ -42,6 +56,7 @@ def check(self):
             make('test-shared')
             make('test-shared-lib')
```

Comment:
It looks like `test-static-lib` was renamed to `test-lib-static` and `test-shared-lib` was renamed to `test-lib-shared. Can you update these? You can just use:
```python
if self.spec.satisfies('@1.4:'):
    make('test-lib-static')
else:
    make('test-static-lib')
```
or something like that.